notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store_optimized.ipynb

```diff
@@ -1227,15 +1227,17 @@
     {
       "cell_type": "code",
       "source": [
-        "from google.cloud.aiplatform_v1beta1.services.feature_online_store_service.transports.grpc import FeatureOnlineStoreServiceGrpcTransport\n",
-        "import grpc\n",
+        "# Uncomment below 2 parts after your PSC setup is completed and replace {endpoint_ip} with your selected one.\n",
```

Comment:
Let's rephrase this as:
"Uncomment the following code blocks after your PSC setup is complete. Replace {endpoint_ip} with the IP of the new connection."